### PR TITLE
Implement MAT-5640 MAT-5658

### PR DESCRIPTION
**Pull request by mcmcphillips**
## MADiE PR

Jira Ticket: [MAT-5640](https://jira.cms.gov/browse/MAT-5640)

(Optional) Related Tickets:
Jira Ticket: [MAT-5658](https://jira.cms.gov/browse/MAT-5658)

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [x] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [x] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

**Comment by RohitKandimalla** [comment]
I am guessing its [https://jira.cms.gov/browse/MAT-5648](https://jira.cms.gov/browse/MAT-5648) rather than 5658

File: src/components/editTestCase/qdm/EditTestCase.tsx [hunk 1]
```diff
@@ -42,28 +72,59 @@ const EditTestCase = () => {
...
+        testCaseTitle={currentTestCase?.title}
+        testCaseId={currentTestCase?.id}
+        measureId={measureId}
       />
+      <form id="edit-test-case-qdm">
+        <div className="allotment-wrapper">
+          <Allotment
+            proportionalLayout={true}
+            separator={true}
+            vertical={false}
```
Comment (RohitKandimalla) [inline src/components/editTestCase/qdm/EditTestCase.tsx:76]: Looks like proportionalLayout and separator defaulted to true, do we still need them? Ignore me if they are not, I am speaking solely based on Allotment docs.

Comment (RohitKandimalla) [reply src/components/editTestCase/qdm/EditTestCase.tsx:76]: also, Do you think the default split should happen 1/3 -2/3? That's what I remember from mock ups

**Review by RohitKandimalla — DISMISSED** [review]

File: src/components/editTestCase/qdm/EditTestCase.tsx (hunk 1 above)
Comment (mcmcphillips) [reply src/components/editTestCase/qdm/EditTestCase.tsx:76]: missed this. updated accordingly

**Review by RohitKandimalla — APPROVED** [review]

**Pull request merged**